### ISSUE TITLE
really good but have a question

### ISSUE THREAD

**Issue by finnonc** [ritz-1337/fortnite-external-evo.gj#4]
fn dumps is not always updated, is there any other way I could get the new offsets?

**Comment by SternI** [comment]
> fn dumps is not always updated, is there any other way I could get the new offsets?

he already give the links to update

**Comment by finnonc** [comment]
> > fn dumps is not always updated, is there any other way I could get the new offsets?
> 
> he already give the links to update

yes but it is from 2 months ago, I did the steps and esp does not work either does aimbot.

**Comment by UniqueApricot8662** [comment]
Dump it yourself, or find them and stop pasting
